plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitSource.java

```diff
@@ -101,6 +98,8 @@
     private final TypeManager typeManager;
     private final Closer closer = Closer.create();
     private final double minimumAssignedSplitWeight;
+    private final TupleDomain<IcebergColumnHandle> enforcedPredicate;
```

Comment:
Maybe rename to `dataColumnPredicate` or `nonMetadataPredicate` to signal that it's a filtered version

Reply:
Renamed to `dataColumnPredicate`.